src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -862,20 +857,15 @@ internal static void DecryptKey(SafeKeyHandle safeKeyHandle, byte[] encryptedDat
         internal static void EncryptKey(SafeKeyHandle safeKeyHandle, byte[] pbKey, int cbKey, bool foep, ref byte[] pbEncryptedKey)
         {
             VerifyValidHandle(safeKeyHandle);
-            if (null == pbKey)
-            {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "pbKey is null"));
-            }
-            if (cbKey < 0)
-            {
-                throw new CryptographicException(SR.Format(SR.Argument_InvalidValue, "cbKey is less than 0"));
-            }
+            Debug.Assert(pbKey != null, SR.Argument_InvalidValue, "pbKey is null");
+            Debug.Assert(cbKey >= 0, SR.Argument_InvalidValue, "cbKey is less than 0");
```

Comment:
Oh, yup, Debug.Assert is correct here; good call.  But the messages should be fixed.  I didn't mention it above, but the length one can be done more helpfully:

`Debug.Assert(cbKey >= 0, $"cbKey is less than 0 ({cbKey});"`